README.md

```diff
@@ -315,7 +314,7 @@ Installing
 - [install-go-ipfs](https://github.com/ipfs/install-go-ipfs) - Install go-ipfs shell script.
 - [install-js-ipfs](https://github.com/ipfs/js-ipfs#install) - Install js-ipfs through npm or a script tag.
 - [ipfs-update](https://github.com/ipfs/ipfs-update) - An updater tool for IPFS.
-- [fs-repo-migrations](https://github.com/ipfs/fs-repo-migrations) - These are migrations for [IPFS fs-repo](https://github.com/ipfs/specs/blob/repo/fs-repo) versions.
+- [fs-repo-migrations](https://github.com/ipfs/fs-repo-migrations) - These are migrations for [IPFS fs-repo](https://github.com/ipfs/specs/tree/399c907b214a24dc82ca010af6884227cb2829cf/repo/fs-repo) versions.
```

Comment:
This probably should be `https://github.com/ipfs/specs/tree/master/repo/fs-repo`.

Reply:
No, I explicitly lock it to a commit as that way, it won't break when/if we move it again. I would argue that most links that we put to specific files and lines on Github should be locked to commits, not pointing to master.